communication/resources/README.md

```diff
@@ -0,0 +1,82 @@
+# Kubernetes Resources
+
+> A collection of resources organized by medium (e.g. audio, text, video)
+
+## Table of Contents
+
+<!-- vim-markdown-toc GFM -->
+
+- [Contributions](#contributions)
+- [Resources](#resources)
+  - [Audio](#audio)
+  - [Text](#text)
+  - [Video](#video)
+  - [Miscellaneous](#miscellaneous)
+
+<!-- vim-markdown-toc -->
+
+## Contributions
+
+If you would like to contribute to this list, please submit a PR and add `/sig contributor-experience` and `/assign @petermbenjamin`.
+
+The criteria for contributions are simple:
+
+- The resource must be related to Kubernetes.
+- The resource must be free.
+
+## Resources
+
+### Audio
+
+- [PodCTL](https://twitter.com/PodCTL)
+- [Kubernetes Podcast](https://kubernetespodcast.com)
+- [The New Stack Podcasts](https://thenewstack.io/podcasts/)
+- [SoundCloud](https://soundcloud.com/search?q=kubernetes)
+
+### Text
+
+- [CNCF Blog](https://www.cncf.io/newsroom/blog/)
+- [Dev.To](https://dev.to/t/kubernetes)
+- [Heptio Blog](https://blog.heptio.com)
+- [KubeTips](http://kubetips.com)
+- [KubeWeekly](https://twitter.com/kubeweekly)
+- [Kubedex](https://kubedex.com/category/blog/)
+- [Kubernetes Blog](https://kubernetes.io/blog/)
+- [Kubernetes Changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md)
+- [Kubernetes Enhancements Repo](https://github.com/kubernetes/enhancements)
+- [Kubernetes Forum](https://discuss.kubernetes.io)
+- [Last Week in Kubernetes Development](http://lwkd.info)
+- [Medium](https://medium.com/tag/kubernetes)
+- [Reddit](https://www.reddit.com/r/kubernetes)
+- [The New Stack Books](https://thenewstack.io/ebooks/)
+- [Weaveworks Blog](https://www.weave.works/blog/category/kubernetes/)
+
+### Video
+
+- [BrightTALK Webinars](https://www.brighttalk.com/search/?q=kubernetes)
+- [Ceph YouTube Channel](https://www.youtube.com/channel/UCno-Fry25FJ7B4RycCxOtfw)
+- [CNCF YouTube Channel](https://www.youtube.com/channel/UCvqbFHwN-nwalWPjPUKpvTA)
+- [CoreOS YouTube Channel](https://www.youtube.com/user/coreoslinux)
```

Comment:
Chances are the content here will become increasingly out of date. I would either qualify as "historical info" or omit.

Reply:
Removed.